package.json

```diff
@@ -125,10 +125,12 @@
     "start:prod": "REACT_APP_ENV=prod node scripts/start.js",
     "build": "node scripts/build.js",
     "test": "node scripts/test.js --env=jsdom",
-    "lint:fix": "eslint --fix --",
+    "lint:fix": "eslint . --fix",
     "extract-messages": "NODE_ENV=production babel ./src  --out-file /dev/null",
     "manage:translations": "NODE_ENV=production ./node_modules/.bin/babel-node scripts/translator.js",
-    "build:langs": "npm run extract-messages && npm run manage:translations"
+    "build:langs": "npm run extract-messages && npm run manage:translations",
+    "precommmit": "npm run lint:fix",
+    "prepush": "npm run lint:fix"
```

Comment:
why not just have travis run `lint:fix`?

Reply:
I prefer do both travis and precommit and prepush, the benefit of using precommit and prepush checking is to make the debug lifecycle shorter. For exmaple, if the style is bad, it will **block developer to do the commit**, therefore developer can know the styling defect as soon as they attempt to commit and recommit again, faster than commit->push->travis->notice failure->update->recommit->repush

Reply:
cool that makes sense to me. but, can't we just tell all our devs to use `lint:fix` before pushing or committing? it just seems redundant to have 3 of the same commands.